microsoft-edge/webview2/concepts/print.md

```diff
@@ -0,0 +1,736 @@
+  CHECK_FAILURE(webView2_16->Print(
+      nullptr, Callback<ICoreWebView2PrintCompletedHandler>(
+          [title = std::move(title), this](HRESULT errorCode, COREWEBVIEW2_PRINT_STATUS printStatus) -> HRESULT
+          {
+            std::wstring message = L"";
+            if (errorCode == S_OK && printStatus == COREWEBVIEW2_PRINT_STATUS_SUCCEEDED)
+            {
+              message = L"Printing " + std::wstring(title.get()) +
+                              L" document to printer is succedded";
+            }
+            else if (errorCode == S_OK && printStatus == COREWEBVIEW2_PRINT_STATUS_PRINTER_UNAVAILABLE)
+            {
+              message = L"Printer is not available, offline or error state";
+            }
+            else if (errorCode == E_ABORT)
+            {
+              message = L"Printing " + std::wstring(title.get()) +
+                                  L" document already in progress";
+            }
+            else
+            {
+              message = L"Printing " + std::wstring(title.get()) +
+                                  L" document to printer is failed";
+            }
+
+            AsyncMessageBox(message, L"Print to default printer");
+
+            return S_OK;
+        })
+        .Get()));
+  return true;
+}
+```
+
+**APIs:**
+
+* [ICoreWebView2_16::Print() method](/microsoft-edge/webview2/reference/win32/icorewebview2_16#print)
+* [COREWEBVIEW2_PRINT_STATUS Enum](/microsoft-edge/webview2/reference/win32/icorewebview2_16#corewebview2_print_status)
+
+---
+
+<!-- end of tab-set -->
+
+### Example 2: Print to printer
+
+This example prints the current web page to a specific printer with the settings.
+
+
+<!-- ------------------------------ -->
+
+# [C#](#tab/csharp)
+
+```csharp
+async void PrintToPrinter()
+{
+  string printerName = GetPrinterName();
+  CoreWebView2PrintSettings printSettings = GetSelectedPrinterPrintSettings(printerName);
+  string title = webView.CoreWebView2.DocumentTitle;
+  try
+  {
+    CoreWebView2PrintStatus printStatus = await webView.CoreWebView2.PrintAsync(printSettings);
+
+    if (printStatus == CoreWebView2PrintStatus.Succeeded)
+    {
+      MessageBox.Show(this, "Printing " + title + " document to printer is succeeded", "Print to printer");
+    }
+    else if (printStatus == CoreWebView2PrintStatus.PrinterUnavailable)
+    {
+      MessageBox.Show(this, "Selected printer is not found, not available, offline or error state", "Print to printer");
+    }
+    else
+    {
+      MessageBox.Show(this, "Printing " + title + " document to printer is failed", "Print");
+    }
+  }
+  catch(ArgumentException)
+  {
+    MessageBox.Show(this, "Invalid settings provided for the specified printer", "Print");
+  }
+  catch (Exception)
+  {
+    MessageBox.Show(this, "Printing " + title + " document already in progress", "Print");
+  }
+}
+
+// Function to get printer name by displaying installed printers list to the user and
```

Comment:
fyi: for now, I'm treating comments in code as out of scope for Writer/Editor pass.

@nishitha-burman - I don't know if this would be an appropriate place to polish comments in code.  If this is copied from another repo upstream, comments should be polished there instead.